ui/app/AppLayouts/Profile/views/MyProfileView.qml

```diff
@@ -319,7 +319,7 @@ SettingsContentBase {
             inShowcaseModel: priv.showcaseModels.accountsVisibleModel
             hiddenModel: priv.showcaseModels.accountsHiddenModel
             showcaseLimit: root.profileStore.getProfileShowcaseEntriesLimit()
-            currentWallet: root.walletStore.overview.mixedcaseAddress
+            currentWallet: !!root.walletStore ? root.walletStore.overview.mixedcaseAddress : ""
```

Comment:
Right, I was about to fix this one too... but there's no `root.walletStore` toplevel property at all defined here :/ This will just silence the warning w/o fixing the problem

Reply:
That was part of an attempt to fix some warnings but I abandoned after going nowhere (mainly `StatusRoundedImage.qml:27:5: QML StatusImage: Cannot open: qrc:/StatusQ/Components/help`). I fixed it now, thanks for the heads up